core/imt/CMakeLists.txt

```diff
@@ -70,6 +71,8 @@ else()
       Core
       MultiProc
   )
+
+  target_sources(Imt PRIVATE src/TExecutor.cxx)
```

Comment:
Shouldn't this go to around line 15, where it's calling `ROOT_LINKER_LIBRARY` outside of the `if` statements?

Reply:
@xvallspl like this it will do the same but will make 1 line of code less :)

Reply:
It could!